### PR TITLE
User profile: hide report abuse link if visiting yourself

### PR DESCRIPTION
**Pull request by rhymes**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [ ] Feature
- [x] Bug Fix
- [ ] Documentation Update

## Description

I was thinking we might want to hide the dropdown for now since it only includes "report abuse" if a user is looking at their own profile

## Added to documentation?

- [ ] docs.dev.to
- [ ] readme
- [x] no documentation needed

### CODE REVIEW

File: spec/requests/user_profile_spec.rb [hunk 1]
```diff
@@ -81,14 +81,6 @@
     end
   end
 
-  describe "GET /user" do
```
Comment (jacobherrington) [inline spec/requests/user_profile_spec.rb]: I'm not sure I understand why this test should be removed in support of the change in `users/show`, would you mind clarifying it (just for my benefit, if nothing else)?

File: app/views/users/show.html.erb [hunk 2]
```diff
@@ -145,14 +145,16 @@
       </div>
     </div>
     <%= render "articles/user_metadata", context: "profile" %>
-    <div class="profile-dropdown">
-      <button id="user-profile-dropdown">
-        <%= image_tag("three-dots.svg", class: "dropdown-icon", alt: "Toggle dropdown menu") %>
-      </button>
-      <div id="user-profile-dropdownmenu" class="profile-dropdownmenu">
-        <a href="/report-abuse?url=https://dev.to/<%= @user.username %>">Report Abuse</a>
+    <% unless @user == current_user %>
```
Comment (jacobherrington) [inline app/views/users/show.html.erb]: Perhaps we could include a TODO comment to move this conditional down to only the report link if more features are added to this dropdown? I'm not sure about your preference when it comes to TODO comments, but I tend to like them.

File: spec/requests/user_profile_spec.rb (hunk 1 above)
Comment (rhymes) [reply spec/requests/user_profile_spec.rb]: Sorry, forgot to explain it's a duplicate of the same test on line 8 https://github.com/thepracticaldev/dev.to/blob/620b72ce4be52cdf694d0fde893c190ee4e44746/spec/requests/user_profile_spec.rb#L8

File: app/views/users/show.html.erb (hunk 2 above)
Comment (rhymes) [reply app/views/users/show.html.erb]: Good idea. I'll add it

File: spec/system/user/view_user_index_spec.rb [hunk 3]
```diff
@@ -85,5 +91,9 @@
         expect(page).to have_link(nil, href: comment.path)
       end
     end
+
+    it "renders the report abuse link" do
```
Comment (lightalloy) [inline spec/system/user/view_user_index_spec.rb]: Maybe "not renders" here? :)

**Comment by rhymes** [comment]
@benhalpern you're right! I'm thinking we should close this PR, it's not the end of the world to have that menu entry and most people won't even notice while looking at their own profile. I noticed because I was reviewing the recent PRs...

**Comment by benhalpern** [comment]
@rhymes yeah I'll close this one because alternate solutions are sort of a different approach altogether.

**Pull request closed without merging**